lib/publiccloud/google.pm

```diff
@@ -61,8 +61,11 @@ sub file2name {
 sub find_img {
     my ($self, $name) = @_;
     my $img_name = $self->file2name($name);
-    return $img_name if (script_output("gcloud compute images list --filter='name~$img_name' |grep -v NAME|grep -v Listed"));
-    return;
+    assert_script_run("echo $img_name");
```

Comment:
If it is only for debugging, maybe `testapi::diag()` is ok too.

Reply:
it was local debug. I forgot to delete it before committing :)